x-pack/plugins/apm/public/components/app/ErrorGroupDetails/index.tsx

```diff
@@ -154,66 +148,56 @@ export function ErrorGroupDetails({ location, match }: ErrorGroupDetailsProps) {
   return (
     <>
       <ErrorGroupHeader groupId={groupId} isUnhandled={isUnhandled} />
-      <EuiPage>
-        <EuiPageBody>
-          <EuiPanel>
-            {showDetails && (
-              <Titles>
-                <EuiText>
-                  {logMessage && (
-                    <Fragment>
-                      <Label>
-                        {i18n.translate(
-                          'xpack.apm.errorGroupDetails.logMessageLabel',
-                          {
-                            defaultMessage: 'Log message',
-                          }
-                        )}
-                      </Label>
-                      <Message>{logMessage}</Message>
-                    </Fragment>
-                  )}
-                  <Label>
-                    {i18n.translate(
-                      'xpack.apm.errorGroupDetails.exceptionMessageLabel',
-                      {
-                        defaultMessage: 'Exception message',
-                      }
-                    )}
-                  </Label>
-                  <Message>{excMessage || NOT_AVAILABLE_LABEL}</Message>
+
+      <EuiPanel>
+        {showDetails && (
+          <Titles>
+            <EuiText>
+              {logMessage && (
+                <Fragment>
```

Comment:
I personally don't like the <Fragment>, but it's up to up   
```suggestion
                <>
```

Reply:
Agree. I was going to fix all of the occurrences but this PR is big already so just fixing this one.